### PR TITLE
feat: Add axios instance

### PR DESCRIPTION
**Pull request by AkshatGKMit**

### CODE REVIEW

File: src/constants/api.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import EnvVars from './env';
```
Comment (adiTrivsme) [inline src/constants/api.ts]: use with alias import

File: src/constants/api.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+const ApiConstants = {
+  BEARER: `Bearer ${EnvVars.API_BEARER_TOKEN}`!,
```
Comment (adiTrivsme) [inline src/constants/api.ts]: use destructured env properties

Comment (adiTrivsme) [inline src/constants/api.ts]: why adding `BEARER` token twice??

File: src/constants/api.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  BASE_URL: EnvVars.API_BASE_URL!,
+  headers: {
+    accept: 'application/json',
+    'content-type': 'application/json',
+    Authorization: EnvVars.API_BEARER_TOKEN!,
```
Comment (adiTrivsme) [inline src/constants/api.ts]: what is `!`?

File: src/network/instance.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import ApiConstants from '@constants/api';
```
Comment (adiTrivsme) [inline src/network/instance.ts]: fix import order `axios` should be imported at top

File: .gitignore [hunk 5]
```diff
@@ -80,4 +80,5 @@ debug.keystore
 .bundle
 .vscode
 __tests__
-.env
\ No newline at end of file
+.env
+DummyScreen.tsx
```
Comment (adiTrivsme) [inline .gitignore]: make sure to remove after testing

File: src/constants/api.ts (hunk 1 above)
Comment (AkshatGKMit) [reply src/constants/api.ts]: I tried with alias but it is not able to access from it.

File: src/constants/api.ts (hunk 3 above)
Comment (AkshatGKMit) [reply src/constants/api.ts]: Using process.env.ANY_Variable. gives `string | undefined` so i added `!`

File: .gitignore (hunk 5 above)
Comment (AkshatGKMit) [reply .gitignore]: Ok

**Review by adiTrivsme — APPROVED** [review]

**Pull request merged**